awslambdaric/lambda_runtime_log_utils.py

```diff
@@ -45,6 +45,10 @@ def from_str(cls, value: str):
         return cls.TEXT.value
 
 
+def _get_log_level_from_env_var(log_level):
+    return {None: "", "TRACE": "DEBUG"}.get(log_level, log_level).upper()
```

Comment:
Why TRACE is mapped to DEBUG?

Reply:
there is no TRACE log level in Python: https://docs.python.org/3/library/logging.html#logging-levels

Reply:
Here I transform `TRACE` to `DEBUG` because Python has no trace level.
And `None` to empty string to make sure that result is always `str` so `str.upper` can be called